drivers/ra/fsp/src/bsp/mcu/all/bsp_rom_registers.c

```diff
@@ -46,7 +46,7 @@
  * Private global variables and functions
  **********************************************************************************************************************/
 
-#if !BSP_CFG_BOOT_IMAGE
+#if !BSP_CFG_BOOT_IMAGE && !defined(CONFIG_SOC_SERIES_RA8P1)
```

Comment:
This is an FSP file, so `BSP_MCU_GROUP_RA8P1` fits better.
```suggestion
#if !BSP_CFG_BOOT_IMAGE && !defined(BSP_MCU_GROUP_RA8P1)
```